build/dockerfiles/Dockerfile.rhel

```diff
@@ -41,9 +42,10 @@ RUN chmod a+rwX            /run/httpd
 RUN mkdir /var/www/html/devfiles
 COPY .htaccess README.md /var/www/html/
 COPY --from=builder /build/devfiles /var/www/html/devfiles
-USER 0
-RUN chmod -R g+rwX /var/www/html/devfiles
 
 STOPSIGNAL SIGWINCH
 
-ENTRYPOINT ["/usr/sbin/httpd", "-D", "FOREGROUND"]
+COPY --from=builder /build/devfiles /var/www/html/devfiles
+COPY ./build/dockerfiles/entrypoint.sh /usr/bin/
```

Comment:
after copying file over again we don't have to fix permissions?

Reply:
The file in the repo has 755 permissions so it's not an issue. I can add the line for explicitness if you prefer, though.

Reply:
Added chmod run to set entrypoint permissions.

Reply:
thanks I just like being explicit because sometimes you pull files on a disrespectful OS (or your GH settings are wack) and you accidentally commit a 644 file when it should be 755. Or so I've heard, it's NEVER happened to me :)